book/04-behind-atom/sections/07-maintaining-your-packages.asc

```diff
@@ -0,0 +1,33 @@
+=== Maintaining Your Packages
+
+While publishing is, by far, the most common action you will perform when working with the packages you provide, there are other things you may need to do.
+
+==== Unpublish a Version
+
+If you mistakenly published a version of your package or perhaps you find a glaring bug or security hole, you may want to unpublish just that version of your package. For example, if your package is named `package-name` and the bad version of your package is v1.2.3 then the command you would execute is:
+
+```bash
+apm unpublish package-name@1.2.3
+```
+
+This will remove just this particular version from the https://atom.io package registry. Anyone who has already downloaded this version will still have it, but it will no longer be available for installation by others.
+
+==== Adding a Collaborator
+
+Some packages get too big for one person. Sometimes priorities change and someone else wants to help out. You can let others help or create co-owners by https://help.github.com/articles/adding-collaborators-to-a-personal-repository/[adding them as a collaborator] on the GitHub repository for your package. *Note:* Anyone that has push access to your repository will have the ability to publish new versions of the package that belongs to that repository.
+
+You can also have packages that are owned by a https://help.github.com/articles/creating-a-new-organization-account/[GitHub organization]. Anyone who is a member of the organization that has push access to the repository for the organization's package will be able to publish new versions of the package.
+
+==== Transferring Ownership
+
+You may one day want to hand off support of your package to someone else. You can do this by simply https://help.github.com/articles/transferring-a-repository/[transferring the GitHub repository] behind your package to the new owner. *Note:* _This is a permanent change! There is no going back!_
```

Comment:
Yes, it is :grinning: I already asked him about it and verified it when the language-treetop package was transferred to me. That's what drove the creation of this documentation.